README.md

```diff
@@ -27,6 +27,7 @@ Twitter: [@vuls_en](https://twitter.com/vuls_en)
 
 | Version     | Main Feature |  Date |
 |:------------|:---------------------------------|:--------------------|
+| [v0.8.0](https://github.com/future-architect/vuls/releases/tag/v0.8.0) | secret | Comming sooln |
```

Comment:
__[Misspell]__ "Comming" is a misspelling of "Coming" ([link](https://sider.review/gh/repos/54808920/pulls/840#issue-4230354))

<sub>posted by Sider</sub>